commands/env-init.help

```diff
@@ -3,7 +3,7 @@
 
 WARDEN_USAGE=$(cat <<EOF
 \033[33mUsage:\033[0m
-  env-init <project_name> <environment_type>
+  env-init <project_name> <environment_type> <environment_version>
```

Comment:
Since `<environment_version>` is optional, it should be surrounded by square brackets: `[<environment_version>]`; otherwise, it appears to be a required argument.